acceptance/tests/direct_puppet/supports_utf8.rb

```diff
@@ -48,22 +53,24 @@
   step 'run agent(s)' do
     with_puppet_running_on(master, {}) do
       agents.each do |agent|
-        fqdn = agent.hostname
```

Comment:
Any reason you are deleting this rather than changing it to `fqdn = agent_to_fqdn(agent)`?

Reply:
Uh because that's what I did when I fixed this problem in the other tests long ago.